ci/test.sh

```diff
@@ -2,4 +2,4 @@
 # Copyright (c) 2015-2017 TNO and Radboud University
 # See LICENSE at root directory of this repository.
 
-cd test/sqatt && stack test --test-arguments="--skip=#long" --work-dir $CACHE_DIR_REL/test/.stack_work --stack-root $CACHE_DIR/.stack --allow-different-user && cd -
+cd test/sqatt && stack test --test-arguments="--skip=#long --skip=#model" --work-dir $CACHE_DIR_REL/test/.stack_work --stack-root $CACHE_DIR/.stack --allow-different-user && cd -
```

Comment:
Do we need to tag all the example names with `#model`? As far as I understand you could instead write `--skip=ExploreModels` flag and it would skip all tests in `ExploreModels`. In this way we avoid the possibility of somebody forgetting to tag an example in `ExploreModels` with `#model`.

Reply:
oh, I didn't think about that! It's definitely better than having #model tag :)

Reply:
`--skip=ExploreModels` doesn't work...